app/assets/javascripts/app/discussions/index.html.slim

```diff
@@ -6,7 +6,7 @@ div ui-view="sidebar"
       .md-toolbar-tools layout="row" layout-align="space-between center"
         a.IconNavigation href="" ng-click="ctrl.toggleSidebar()"
         a href="#/discussions" Council
-        a href="#/members" Members
+        a href="#"
```

Comment:
It was a 'hack' for the flex box maintain the logo centered but I have found a better way, I´ll change that